docs/rules.md

```diff
@@ -50,6 +50,7 @@
 * [`no-missing-eof-newline`](../src/rules/no-missing-eof-newline/README.md): Disallow missing end-of-file newline.
 * [`no-multiple-empty-lines`](../src/rules/no-multiple-empty-lines/README.md): Disallow multiple empty lines.
 * [`number-leading-zero`](../src/rules/number-leading-zero/README.md): Require or disallow a leading zero for fractional numbers less than 1.
+* [`number-max-precision`](../src/rules/number-max-precision/README.md): Limit the number of decimal places allowed in numbers.
```

Comment:
:+1: I like the use of "Limit" here instead of "Specify", we should use it to start all the `*-max-*` rules :)